app/controllers/fae/utilities_controller.rb

```diff
@@ -54,7 +54,25 @@ def records_by_display_name(query)
       all_models.each do |m|
         records += m.fae_search(query)
       end
-      records
+      authorize_records(records)
+    end
+
+    def authorize_records(records)
```

Comment:
that's originally the route i went with this, then realized it doesn't work for situations where  users are limited from seeing arbitrary fae::pages, ill take another look in the AM and see what might be possible here